modules/FP_events/language/en_us.lang.php

```diff
@@ -134,5 +134,6 @@
 	'LBL_NAME_SECOND' => 'Last Name',
      
 	'LBL_ACTIVITY_STATUS' => 'Activity Status',
+	'LBL_FP_EVENT_LOCATIONS_FP_EVENTS_1_FROM_FP_EVENTS_TITLE' => 'Locations',
```

Comment:
Please include the comment after the commented line so translators can see it as a suggestion directly on Crowdin. 
//LBL_ERROR_MSG_4 Begins with a number (controller.php line 581) for example 10 emails have failed to send. 

If done it will look like this:
![2016-11-24 11_41_07-all strings - suitecrm core translation - crowdin translation e mais 1 pagina -](https://cloud.githubusercontent.com/assets/3055443/20597349/fb8d01fa-b23a-11e6-9682-305bc2479046.png)